.gitmodules

```diff
@@ -29,3 +29,6 @@
 	path = external/llvm
 	url = https://github.com/mono/llvm.git
 	branch = master
+[submodule "external/xamarin-android-tools"]
+	path = external/xamarin-android-tools
+	url = git@github.com:xamarin/xamarin-android-tools.git
```

Comment:
This should be an `https://` url, like the other submodules. Requiring `git@` urls complicates checkouts for those who don't have GitHub accounts, iirc. (Or something like that...?)